dir.props

```diff
@@ -48,6 +48,12 @@
     <ToolsDir Condition="'$(UseToolRuntimeForToolsDir)'=='true'">$(ToolRuntimePath)</ToolsDir>
     <ToolsDir Condition="'$(ToolsDir)'==''">$(ProjectDir)Tools/</ToolsDir>
     <ToolRuntimePath Condition="'$(ToolRuntimePath)'==''">$(ToolsDir)</ToolRuntimePath>
+    
+    <!-- Temporarily enable local build of tools -->
```

Comment:
Some props file under the "Tools-Override" directory that gets imported from the Tools directory is what I had in mind.